teos/cli/teos_cli.py

```diff
@@ -54,36 +83,66 @@ def main(command, args, command_line_conf):
 
     setup_data_folder(DATA_DIR)
 
+    teos_rpc_host = config.get("RPC_BIND")
+    teos_rpc_port = config.get("RPC_PORT")
+
+    rpc_client = RPCClient(teos_rpc_host, teos_rpc_port)
+
     try:
         if command == "get_all_appointments":
-            appointment_data = get_all_appointments(config.get("RPC_BIND"), config.get("RPC_PORT"))
-            if appointment_data:
-                print(json.dumps(appointment_data, indent=4, sort_keys=True))
+            result = rpc_client.get_all_appointments()
+
+        elif command == "get_tower_info":
+            result = rpc_client.get_tower_info()
+
+        elif command == "get_users":
+            result = rpc_client.get_users()
+
+        elif command == "get_user":
+            if not args:
+                sys.exit("No user_id was given")
+            if len(args) > 1:
+                sys.exit(f"Expected only one argument, not {len(args)}")
 
+            result = rpc_client.get_user(args[0])
         elif command == "help":
             if args:
                 command = args.pop(0)
 
                 if command == "get_all_appointments":
                     sys.exit(help_get_all_appointments())
 
+                elif command == "get_tower_info":
+                    sys.exit(help_get_tower_info())
+
+                elif command == "get_users":
+                    sys.exit(help_get_users())
+
+                elif command == "get_user":
+                    sys.exit(help_get_user())
+
                 else:
                     sys.exit("Unknown command. Use help to check the list of available commands")
 
             else:
                 sys.exit(show_usage())
 
+    except grpc.RpcError as e:
+        sys.exit(e.details())
     except (FileNotFoundError, IOError, ConnectionError, ValueError) as e:
         sys.exit(str(e))
     except (InvalidKey, InvalidParameter, TowerResponseError, SignatureError) as e:
         sys.exit(f"{e.msg}. Error arguments: {e.kwargs}")
     except Exception as e:
         sys.exit(f"Unknown error occurred: {str(e)}")
 
+    if result:
```

Comment:
Kinda nitpicking here, but would't this be better off before all the exceptions? In terms of readability basically. Also result may be not defined, but I don't think there is any care where the flow won't have been forced to exit before that.

Reply:
I moved that inside the `try` block, I agree it's more readable. The reason I put the `if` is that in case we later have a command that can return empty output (but not because of failure) I didn't want to print an empty line.